consensus_pbft.go

```diff
@@ -741,9 +751,9 @@ func (p *Pbft) getNextMessage(span trace.Span) (*MessageReq, bool) {
 			p.spanAddEventMessage("dropMessage", span, msg)
 		}
 		if msg != nil {
+			p.logger.Printf("[TRACE] Received %s", msg)
```

Comment:
Why did you move this?

Reply:
Just wanted to make it consistent with the for loop above: https://github.com/0xPolygon/pbft-consensus/pull/76/files#diff-b208c241c10c010125c6a1c4f43ac407c5fb7b363bab10a050e5681019875478R750-R751. There we are first logging and than reporting metadata to the tracer, so wanted to keep it the same in this block as well.

I can revert it back,  if it bothers you.